build.sbt

```diff
@@ -0,0 +1,19 @@
+ThisBuild / fork in Test := true
+
+ThisBuild / scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings")
+
+ThisBuild / organization := "com.socrata"
+
+ThisBuild / resolvers += "socrata" at "https://repo.socrata.com/artifactory/libs-release"
+
+val commonPG = project in file("common-pg")
```

Comment:
Not particularly.  The difference is that, just as in ordinary Scala code, a `lazy val` can (semi-)safely refer to things defined after it since it doesn't try to evaluate until requested.